### ISSUE TITLE
0.4.1 does not start on Windows

### ISSUE THREAD

**Issue by xaozai** [yggdrasil-network/yggdrasil-go#862]
Windows Server 2012 R2 6.3 (9600) x64

```
C:\Windows\system32>"C:\Program Files\Yggdrasil\yggdrasil.exe" -useconffile C:\yggdrasil\yggdrasil.conf
2021/11/03 21:36:44 Build name: yggdrasil
2021/11/03 21:36:44 Build version: 0.4.1
2021/11/03 21:36:44 Starting up...
2021/11/03 21:36:44 Startup complete
2021/11/03 21:36:44 Starting multicast module
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x20 pc=0x54959d]

goroutine 27 [running]:
golang.org/x/sys/windows.UTF16PtrToString(0x0)
        C:/Users/appveyor/go/pkg/mod/golang.org/x/sys@v0.0.0-20211102192858-4dd7
2447c267/windows/syscall_windows.go:132 +0x1d
golang.zx2c4.com/wireguard/tun/wintun.logMessage(0x0, 0x27e6e4a0, 0x0)
        C:/Users/appveyor/go/pkg/mod/golang.zx2c4.com/wireguard@v0.0.0-202110170
52713-f87e87af0d9a/tun/wintun/wintun_windows.go:49 +0xec
syscall.Syscall(0x7ff9b3cb6120, 0x3, 0xc00019ef30, 0xc00019ef48, 0xc000287628)
        C:/go/src/runtime/syscall_windows.go:479 +0xf4
golang.zx2c4.com/wireguard/tun/wintun.CreateAdapter({0xc00018c080, 0xc0002875b8}
, {0x6d9043, 0x9}, 0xc000287628)
        C:/Users/appveyor/go/pkg/mod/golang.zx2c4.com/wireguard@v0.0.0-202110170
52713-f87e87af0d9a/tun/wintun/wintun_windows.go:86 +0x10d
golang.zx2c4.com/wireguard/tun.CreateTUNWithRequestedGUID({0xc00018c080, 0xa7d0e
9c41f06ebb2}, 0x6e62c5, 0xcf39)
        C:/Users/appveyor/go/pkg/mod/golang.zx2c4.com/wireguard@v0.0.0-202110170
52713-f87e87af0d9a/tun/tun_windows.go:71 +0x46
github.com/yggdrasil-network/yggdrasil-go/src/tuntap.(*TunAdapter).setup.func1()

        C:/projects/yggdrasil-go/src/tuntap/tun_windows.go:34 +0xb1
golang.zx2c4.com/wireguard/windows/elevate.DoAsSystem(0xc000287ac8)
        C:/Users/appveyor/go/pkg/mod/golang.zx2c4.com/wireguard/windows@v0.4.12/
elevate/doas.go:114 +0x622
github.com/yggdrasil-network/yggdrasil-go/src/tuntap.(*TunAdapter).setup(0x6d81a
1, {0xc00018c080, 0xc0001f7ca0}, {0xc0001de120, 0x2}, 0x0)
        C:/projects/yggdrasil-go/src/tuntap/tun_windows.go:27 +0x176
github.com/yggdrasil-network/yggdrasil-go/src/tuntap.(*TunAdapter)._start(0xc000
092000)
        C:/projects/yggdrasil-go/src/tuntap/tun.go:135 +0x2b4
github.com/yggdrasil-network/yggdrasil-go/src/tuntap.(*TunAdapter).Start.func1()

        C:/projects/yggdrasil-go/src/tuntap/tun.go:108 +0x26
github.com/Arceliar/phony.Block.func1()
        C:/Users/appveyor/go/pkg/mod/github.com/!arceliar/phony@v0.0.0-202102092
35338-dde1a8dca979/actor.go:80 +0x26
github.com/Arceliar/phony.(*Inbox).run(0xc000092048)
        C:/Users/appveyor/go/pkg/mod/github.com/!arceliar/phony@v0.0.0-202102092
35338-dde1a8dca979/actor.go:89 +0x34
created by github.com/Arceliar/phony.(*Inbox).restart
        C:/Users/appveyor/go/pkg/mod/github.com/!arceliar/phony@v0.0.0-202102092
35338-dde1a8dca979/actor.go:121 +0x5b
```

**Comment by neilalexander** [comment]
This looks like a problem in the Wireguard TUN library itself?

**Comment by xaozai** [comment]
> This looks like a problem in the Wireguard TUN library itself?

I don't know...

0.4 works fine. Rolled back.

**Comment by neilalexander** [comment]
Please try this instead:

64-bit: https://ci.appveyor.com/api/buildjobs/tjcu1uek537gwmjb/artifacts/yggdrasil-develop-0.4.1-0001-x64.msi
32-bit: https://ci.appveyor.com/api/buildjobs/tjcu1uek537gwmjb/artifacts/yggdrasil-develop-0.4.1-0001-x86.msi

**Comment by xaozai** [comment]
> Please try this instead:
> 
> 64-bit: https://ci.appveyor.com/api/buildjobs/tjcu1uek537gwmjb/artifacts/yggdrasil-develop-0.4.1-0001-x64.msi 32-bit: https://ci.appveyor.com/api/buildjobs/tjcu1uek537gwmjb/artifacts/yggdrasil-develop-0.4.1-0001-x86.msi

Wow, so fast )
I tried x64 -  it started and it seems to be working fine.
Thanks! :+1 : )

**Comment by neilalexander** [comment]
Should be fixed now with v0.4.2.

**Comment by r4sas** [comment]
Everything works correctly with 0.4.1, just someone forgot to update library included to installer.
